src/backend/utils/adt/jsonpath_exec.c

```diff
@@ -526,13 +535,27 @@ tsql_openjson_with_columnize(Jsonb *jb, char *col_info)
 			asjson;
 	col_path = NULL; col_type = NULL; strict = false; as = false; asjson = false;
 	token = strtok(col_info, " ");
+
+	if (strncmp(token, "strict", 6) == 0)
```

Comment:
Is it possible to have lower case and upper case issue in here ?

Reply:
If not , pls add a test case of STRICT upper case

Reply:
Mixed case currently handled properly, will add test case
```
1> SELECT * FROM OPENJSON('{"obj":{}}') WITH(field int 'STRICT $.obj')
2> GO
Msg 33557097, Level 16, State 1, Server BABELFISH, Line 1
syntax error at end of jsonpath input
1> SELECT * FROM OPENJSON('{"obj":{}}') WITH(field int 'StRiCT $.obj')
2> go
Msg 33557097, Level 16, State 1, Server BABELFISH, Line 1
syntax error at end of jsonpath input
```